### ISSUE TITLE
"Cancel Sign Up" Button

### ISSUE THREAD

**Issue by jhulen** [hack4impact-utk/rooted-east#138]
As a participant, I want to be able to cancel my sign-up for an event when I am feeling unsocial.

For this story, you are focused on the cancel sign up button on the Events page. Under "Your Events," any event that is in the future (not ones in the past, ofc) should have the option to un-sign-up. 

When a person clicks this button (they can click it from "All Upcoming Events" component or from "Your Events" component) there will be a pop up window that says "You are no longer signed up." (If you are feeling fancy, you can add the event name in the pop-up, but this isn't necessary) the pop up window either closes after a couple seconds, or has an "x" out option in the corner. 

TL;DR
1. create "cancel sign up" button for any event a participant is signed up for.
2. pop up window says "you are no longer signed up" or something.
3. Remove event from "Your Events" and update it in the database.

you got this, lil fella. 

<img width="683" alt="Screenshot 2024-09-30 at 3 50 37 PM" src="https://github.com/user-attachments/assets/d848ef1a-bc7e-4a19-9ec2-56fbe9218e6b">
